src/main/java/model/Initializer.java

```diff
@@ -0,0 +1,66 @@
+package model;
+
+import Service.ComService;
+import Service.LibrSerializer;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import description.Commands;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+
+import java.util.Map;
+import java.util.Scanner;
+import java.util.function.Consumer;
+
+public class Initializer {
+    private final static Map<String, Consumer<Scanner>> commands = new HashMap();
+
+    private static LibrSerializer librSerializer = new LibrSerializer(new JsonMapper());
+
+    private static ComService comService= new ComService();
+
+    private static  Library libr;
+    static {
+        initializer();
+        try {
+            libr = librSerializer.getLib();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void main(String[] args) {
+        Scanner scanner=new Scanner(System.in);
+        System.out.println("Your name: ");
+        String command = scanner.nextLine().toLowerCase();
+        System.out.println("write help if you need help: ");
+
+        while (!command.equals("exit")){
+            command = scanner.nextLine().toLowerCase();
+            commands.getOrDefault(command,
+                    scanner1 -> System.out.println("incorrect command")).accept(scanner);
+        }
+    }
+
+    private static void initializer() {
+        commands.put(Commands.HELP.value, scanner -> Arrays.stream(Commands.values()).forEach(System.out::println));
+        commands.put(Commands.EXIT.value, scanner -> System.out.println("exit"));
+        commands.put(Commands.SHOW.value, comService.getShowCommand(libr));
+        commands.put(Commands.SAVE.value, comService.getSaveCommand(libr, librSerializer));
+        commands.put(Commands.ADD.value, scanner -> {
```

Comment:
why comService not returning add command? you have some command like save and show from comService and others not, it inconsistent, move  them to service or move them all out of the service